src/components/button/index.tsx

```diff
@@ -1,10 +1,9 @@
 import _ from 'lodash';
+import memoize from 'memoize-one';
 import React, {PureComponent} from 'react';
-import {Platform, StyleSheet, LayoutAnimation, LayoutChangeEvent, ImageStyle} from 'react-native';
```

Comment:
I did not yet look at all the code because I'm getting worse performance in the `Playground.perf` :/
How are you testing this change?

Reply:
I checked again with the button screen, I run the tests 3 times for each scenario (means 30 for each in total):
- on master - an average of 74.3ms
- on this PR - an average of 71.1ms 
I agree it's a minor improvement, but at least it's not getting worst :)